daprdocs/content/en/developing-applications/integrations/gRPC-integration.md

```diff
@@ -15,7 +15,7 @@ You can find a list of auto-generated clients [here](https://github.com/dapr/doc
 
 The Dapr runtime implements a [proto service](https://github.com/dapr/dapr/blob/master/dapr/proto/runtime/v1/dapr.proto) that apps can communicate with via gRPC.
 
-In addition to calling Dapr via gRPC, Dapr can communicate with an application via gRPC. To do that, the app needs to host a gRPC server and implements the [Dapr appcallback service](https://github.com/dapr/dapr/blob/master/dapr/proto/runtime/v1/appcallback.proto)
+In addition to calling Dapr via gRPC, Dapr supports service to service calls with gRPC by acting as a proxy. See more information [here]({{< ref howto-invoke-services-grpc.md >}}).
```

Comment:
Still not clear how this article relates to this new one and when to use the --app-protocol setting

Reply:
There's only one way to use gRPC and Dapr and this is it - there isn't any other way. you still need the app-protocol when Dapr is fronting a gRPC service, that hasn't changed and it still in this how-to.

Reply:
"that hasn't changed and it still in this how-to." - Where does the --app-protocol come into this HowTo?

Reply:
Search for --app-protocol in the file and see.